docker-compose.yml

```
@@ -0,0 +1,25 @@
+version: '3.8'
+
+services:
+
+  mysql_server:
+    image: mariadb:latest
+    restart: always
+    environment:
+      - MARIADB_ALLOW_EMPTY_ROOT_PASSWORD=true
+      - MYSQL_DATABASE=fullstack_shop
```

To mozesz z envow wczytac:
```suggestion
      - MYSQL_DATABASE=${DB_DATABASE}
```
[Tutaj](https://docs.docker.com/compose/environment-variables/) masz link do docsow, gdzie mozesz doczytac szczegoly.

---

I was trying to do it that way, but it didn't work.